modules/programs/process-compose.nix

```
@@ -0,0 +1,51 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  inherit (lib) literalExpression mkIf;
+  cfg = config.programs.process-compose;
+  settingsFormat = pkgs.formats.yaml { };
+in
+{
+  meta.maintainers = [ lib.hm.maintainers.bbigras ];
+  options.programs.process-compose = {
+    enable = lib.mkEnableOption "the process-compose orchestrator";
+    package = lib.mkPackageOption pkgs "process-compose" { nullable = true; };
+    settings = lib.mkOption {
+      type = settingsFormat.type;
+      default = { };
+      description = ''
+        The process-compose configuration.
+        See [docs](`https://f1bonacc1.github.io/process-compose/tui/?h=settings.yaml#tui-state-settings`).
```

shouldn't it be this?
```suggestion
        See [docs](`https://f1bonacc1.github.io/process-compose/configuration/`).
```

---

I'm interested in generating the `settings.yaml` file, which https://f1bonacc1.github.io/process-compose/configuration/ doesn't seem to mention.

---

oh I see, why don't you add support for `shortcuts.yaml` and `process-compose.yml`?